src/plugin/controller.ts

```diff
@@ -15,6 +15,14 @@ let borderRadiusArray = [0, 8, 12, 16];
 let originalNodeTree = [];
 let lintVectors = false;
 
+figma.on("documentchange", _event => {
+  // When a change happens in the document
+  // send a message to the plugin to look for changes.
+  figma.ui.postMessage({
+    type: "change"
+  });
+});
+
```

Comment:
@pabloserrago This is the bit that enables automatic feedback. It was removed, but I added it back.